tests/network/bindingplugin.go

```diff
@@ -100,6 +104,65 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(vmi.Status.Interfaces[0].MAC).To(Equal(macAddress))
 		})
 	})
+
+	Context("macvtap", func() {
+		const (
+			macvtapNetworkConfNAD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
+			macvtapBindingName    = "macvtap"
+			macvtapLowerDevice    = "eth0"
+			macvtapNetworkName    = "net1"
+		)
+
+		createMacvtapNetworkAttachmentDefinition := func(namespace, networkName, macvtapLowerDevice string) error {
+			macvtapNad := fmt.Sprintf(macvtapNetworkConfNAD, networkName, namespace, macvtapLowerDevice, networkName)
+			return createNetworkAttachmentDefinition(kubevirt.Client(), networkName, namespace, macvtapNad)
+		}
+
+		BeforeEach(func() {
+			Expect(createMacvtapNetworkAttachmentDefinition(testsuite.GetTestNamespace(nil), macvtapNetworkName, macvtapLowerDevice)).
+				To(Succeed(), "A macvtap network named %s should be provisioned", macvtapNetworkName)
+		})
+
+		BeforeEach(func() {
+			err := libkvconfig.WithNetBindingPlugin(macvtapBindingName, v1.InterfaceBindingPlugin{DomainAttachmentType: v1.Tap})
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		Context("can run a virtual machine with one macvtap interface", func() {
```

Comment:
I do not see a need for another indented context here. We are in the "macvtap" context, and if there is a need for some pre-setup, it can be done at that level.

But I also think it is wrong to keep the current separation between the fixture and test body.
The test itself is also about defining the VM correctly.

Reply:
done.